foundry.toml

```diff
@@ -4,9 +4,12 @@ optimizer_runs = 800
 via_ir = false
 ffi = true
 fs_permissions = [{ access = "read-write", path = ".forge-snapshots/"}, { access = "read", path = "./out"}]
-evm_version = 'paris'
+cancun = true
+solc = "./bin/solc"
 
 [profile.ci]
 fuzz_runs = 100000
+solc = "./bin/solc-static-linux"
```

Comment:
wait so does the other one only work on mac? thats awk for people who run non-mac locally

Reply:
Yeah there are different binaries for diff machines :/

Reply:
what if we made .env set the solc binary environment variable depending on os?
something like
```
if [[ "$OSTYPE" == "linux-gnu"* ]]; then
        FOUNDRY_SOLC = ...
elif [[ "$OSTYPE" == "darwin"* ]]; then
        FOUNDRY_SOLC = ...
else
        # Unknown.
fi
```

Reply:
windows users have questions?

Reply:
if youre a windows user can you `echo $OSTYPE` so we know what to put? I _think_ its "win32" but not sure